### PR TITLE
Migrate stored trigger view to new scheme

### PR DESCRIPTION
**Pull request by jnwatson**
In storage, each trigger contains the iden of the view the trigger belongs to. In #1402, the view iden for the default view changed. On startup, check for any stored triggers whose view iden is the same as the cortex iden, and change it to the new view iden.

### CODE REVIEW

File: synapse/cortex.py [hunk 1]
```diff
@@ -751,6 +751,7 @@ class Cortex(s_cell.Cell):
         self.stormcmds = {}
         self.stormvars = None  # type: s_hive.HiveDict
         self.stormrunts = {}
+        self.stormdmons = {}
```
Comment (vEpiphyte) [inline synapse/cortex.py]: ????

File: synapse/lib/trigger.py [hunk 2]
```diff
@@ -264,6 +264,8 @@ def migrate_v0_rules(self):
         Notes:
             v0 had two differences user was a username.  Replaced with iden of user as 'iden' field.
             Also 'iden' was storage as binary.  Now it is stored as hex string.
+
+        TODO:  due to our migration policy, remove in 0.2.0
```
Comment (vEpiphyte) [inline synapse/lib/trigger.py]: docstring -> comment

File: synapse/tests/test_lib_trigger.py [hunk 3]
```diff
@@ -41,11 +41,20 @@ class TrigTest(s_t_utils.SynTest):
...
+                self.eq(iden2, s_common.ehex(v0iden))
                 self.eq(trig2.ver, 1)
                 self.eq(trig2.storm, 'testcmd')
                 self.eq(trig2.useriden, rootiden)
 
+    async def test_view_migration(self):
+        '''
+        Make sure the trigger's view was migrated from iden=cortex.iden to its own
+        '''
+        async with self.getRegrCore('0.1.0-trigger') as core:
```
Comment (vEpiphyte) [inline synapse/tests/test_lib_trigger.py]: was this cortex made with 0.1.0 or 0.1.34?

**Comment by jnwatson** [comment]
0.1.32

**Comment by jnwatson** [comment]
The lack of this was causing an exception in cortex fini.

File: synapse/tests/test_lib_trigger.py (hunk 3 above)
Comment (vEpiphyte) [reply synapse/tests/test_lib_trigger.py]: Can we rename this to `0.1.32-trigger` then?

**Comment by jnwatson** [comment]
done.

**Pull request merged**